src/components/FirstBanner/index.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useState } from "react";
+import niceView from "../../images/nice_view.jpeg";
+import Image from "next/image";
+import styled from "styled-components";
+
+const Title = styled.h1`
```

Comment:
Try separating all these styled-components to a new file and import it as `import * as S from './styles'`